x-pack/plugins/transform/public/app/app_dependencies.tsx

```diff
@@ -48,6 +49,7 @@ export interface AppDependencies {
   ml: GetMlSharedImportsReturnType;
   spaces?: SpacesPluginStart;
   triggersActionsUi: TriggersAndActionsUIPublicPluginStart;
+  savedObjectTagging?: SavedObjectTaggingPluginStart;
```

Comment:
NIT: would rather use the underlying `SavedObjectsTaggingApi` type instead instead of the non-OSS alias.